unittests/rem_voting_test.cpp

```diff
@@ -510,20 +510,20 @@ BOOST_FIXTURE_TEST_CASE( rem_vote_weight_test, voting_tester ) {
          // rem weight:      0.678;
```

Comment:
update "weeks to mature" and "rem weight" in the comments

Reply:
Done.